packages/gatsby-adapter-netlify/src/__tests__/route-handler.ts

```diff
@@ -143,10 +145,19 @@ describe(`route-handler`, () => {
     })
   })
 
-  describe(`forceRedirects`, () => {
-    const routesManifest = []
-    await handleRoutesManifest(routesManifest)
-
-    expect(status).toEqual(`200!`)
-  })
+  // describe(`forceRedirects`, () => {
+  //   it(`honors the force parameter`, async () => {
+  //     const redirects: Route = {
+  //       path: `/old-url`,
+  //       type: `redirect`,
+  //       toPath: `/new-url`,
+  //       status: 200,
+  //       headers: [{}],
+  //       force: true,
+  //       conditions: { language: [`ca`, `us`] },
+  //     }
+
+  //     await handleRoutesManifest([redirects])
```

Comment:
Still trying to figure out how to test this, since `handleRoutesManifest` doesn't return anything to check against. Using the debugger, I can see the status being set to `200!` within the method (and also parsing the conditions 🎉), but can't read the output `injectEntries` writes to `public/_redirects` from the test.

Reply:
Let's split `handleRoutesManifest` into 2 functions - one that does most of the work and returns the `_redirect` and `_headers` content which we can assert and the other one (probably will keep existing name) that calls the new function and then call `injectEntries` with results

That would allow for easy assertion (without creating some kind of spy, mocks or allowing function to actually write the files out and then read that file back).

Does that sound fine?